.github/workflows/log-viewer-docs.yaml

```
@@ -0,0 +1,57 @@
+name: "log-viewer-docs"
```

```suggestion
name: "docs"
```

Should be sufficient since there are no other docs workflows right now.